alteryx/woodwork
# v0.0.3

**v0.0.3** October 9, 2020
- Enhancements
    - Implement setitem on DataTable to create/overwrite an existing DataColumn (#165)
    - Add ``to_pandas`` method to DataColumn to access the underlying series (#169)
    - Add list_logical_types function and CLI to get dataframe of woodwork LogicalTypes (#172)
    - Add ``describe`` method to DataTable to generate statistics for the underlying data (#181)
    - Add optional ``return_dataframe`` parameter to ``load_retail`` to return either DataFrame or DataTable (#189)
    - Add ``get_mutual_information`` method to DataTable to generate mutual information between columns (#203)
    - Add ``read_csv`` function to create DataTable directly from CSV file (#222)
- Fixes
    - Fix bug causing incorrect values for quartiles in ``DataTable.describe`` method (#187)
    - Fix bug in ``DataTable.describe`` that could cause an error if certain semantic tags were applied improperly (#190)
    - Fix bug with instantiated LogicalTypes breaking when used with issubclass (#231)
- Changes
    - Remove unnecessary ``add_standard_tags`` attribute from DataTable (#171)
    - Remove standard tags from index column and do not return stats for index column from ``DataTable.describe`` (#196)
    - Update ``DataColumn.set_semantic_tags`` and ``DataColumn.add_semantic_tags`` to return new objects (#205)
    - Update various DataTable methods to return new objects rather than modifying in place (#210)
    - Move datetime_format to Datetime LogicalType (#216)
    - Do not calculate mutual info with index column in ``DataTable.get_mutual_information`` (#221)
    - Move setting of underlying physical types from DataTable to DataColumn (#233)
- Documentation Changes
    - Remove unused code from sphinx conf.py, update with Github URL(#160`, :pr:`163)
    - Update README and docs with new Woodwork logo, with better code snippets (#161`, :pr:`159)
    - Add DataTable and DataColumn to API Reference (#162)
    - Add docstrings to LogicalType classes (#168)
    - Add Woodwork image to index, clear outputs of Jupyter notebook in docs (#173)
    - Update contributing.md, release.md with all instructions (#176)
    - Add section for setting index and time index to start notebook (#179)
    - Rename changelog to Release Notes (#193)
    - Add section for standard tags to start notebook (#188)
    - Add Understanding Types and Tags user guide (#201)
    - Add missing docstring to ``list_logical_types`` (#202)
    - Add Woodwork Global Configuration Options guide (#215)
- Testing Changes
    - Add tests that confirm dtypes are as expected after DataTable init (#152)
    - Remove unused ``none_df`` test fixture (#224)
    - Add test for ``LogicalType.__str__`` method (#225)
